centaur/src/main/resources/standardTestCases/workflow_engine_functions/workflow_engine_functions.wdl

```diff
@@ -0,0 +1,24 @@
+version 1.0
+
+
+workflow w {
+  String lines1 = "hello"
+  String lines2 = "world"
+  String map1 = "foo\tbar"
+  String map2 = "baz\tqux"
+  Array[String] array_lines = [lines1, lines2]
+  Array[String] array_map = [map1, map2]
+  File array_file = write_lines(array_lines)
+  File map_file = write_lines(array_map)
+
+  Array[String] out_array = read_lines(array_file)
+  Map[String, String] out_map = read_map(map_file)
```

Comment:
These two variations seem to be testing more the underlying code in `read_map` rather than the ability to call it from a workflow context?

Admittedly I'm assuming that they're ultimately calling into the same IO function so to me they look the same... Or are they genuinely implemented differently under the hood and worth checking separately?

Reply:
My goal here was only to create a rough equivalent to the CTKS test I deleted, though I agree it would be a good idea to have more rigorous testing around workflow-context functions.